insights/parsers/getconf_pagesize.py

```diff
@@ -0,0 +1,32 @@
+"""
+getconf PAGE_SIZE - command ``/usr/sbin/getconf PAGE_SIZE``
+===========================================================
+
+This very simple parser returns the output of the ``getconf PAGE_SIZE`` command.
+
+Examples:
+
+    >>> pagesize_parsed.page_size
+    '4096'
+"""
+from . import ParseException
+from .. import parser, CommandParser
+from insights.specs import Specs
+
+
+@parser(Specs.getconf_page_size)
+class GetconfPageSize(CommandParser):
+    """Class for parsing 'getconf PAGE_SIZE' command output
+
+    Output: page_size
+    """
```

Comment:
Since the `page_size` is a valuable attribute provided by this parser, better add `Attributes` docstring for it.

For details, please see: https://www.sphinx-doc.org/en/1.8/usage/extensions/example_google.html?highlight=Attributes.
Thanks.

Reply:
For example,
```
Attributes:
    page_size (int): integer value of the current configuration page size.
```